src/ansible_compat/runtime.py

```diff
@@ -461,6 +466,22 @@ def _install_galaxy_role(
             link_path,
         )
 
+    def _update_env(self, varname: str, value: List[str], default: str = "") -> None:
+        """Update colon based environment variable if needed.
+
+        New values are added by inserting them to assure they take precedence
+        over the existing ones.
+        """
+        if value:
+            orig_value = self.environ.get(varname, default)
+            if orig_value:
+                # Prepend original or default variable content to custom content.
+                value = [*value, *orig_value.split(':')]
+            value_str = ":".join(value)
+            if value_str != self.environ.get(varname, ""):
+                self.environ[varname] = value_str
+                _logger.info("Added %s=%s", varname, value_str)
```

Comment:
This message is misleading since adding variables is just one of the possibilities here. The other one is pretending information to the existing value.

Reply:
I will use "Set" instead of "Added"